### DISCUSSION TITLE
Parentheses autocomplete for arrow functions?

### DISCUSSION THREAD

**Discussion by Si1adan** [microsoft/vscode-discussions#400] [Extension Development QnA] [answered]
Is there any way to turn on autocompletion with parentheses for arrow functions?
I already use ```javascript.suggest.completeFunctionCalls```, but it works only for regular functions.

```javascript
function foo () {
}

const bar = () => {
}

foo();  // autocompleted ()

bar
```

**Comment by isidorn** [comment, marked as answer]
Not sure. Sounds like a good question for stack overflow. This repository is a discussion place for VS Code Extension Authors.
